lib/location_info/tokenizer_mixin.js

```diff
@@ -82,8 +82,8 @@ exports.assign = function (tokenizer) {
         attachLocationInfo(this.currentToken);
     };
 
-    tokenizer._createDoctypeToken = function (doctypeNameFirstCh) {
-        tokenizerProto._createDoctypeToken.call(this, doctypeNameFirstCh);
+    tokenizer._createDoctypeToken = function (initializeName) {
```

Comment:
Ah, it's because it's a boolean and `initialName` would be misleading as it could suggest it's a string.